automation/src/test/scala/org/broadinstitute/dsde/automation/config/AuthTokens.scala

```diff
@@ -1,12 +1,11 @@
-package org.broadinstitute.dsde.firecloud.config
+package org.broadinstitute.dsde.automation.config
 
 import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
 import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
 import com.google.api.client.json.jackson2.JacksonFactory
 
 import scala.collection.JavaConverters._
 
-
 object AuthTokens {
```

Comment:
these are kind of firecloud-specific, aren't they?  Specific to our testing framework, at least

Reply:
Maybe I'm using the wrong words to describe the separation I'm looking for.

I do want to keep things that are specific to our testing framework but useful for all Workbench services, like FIAB users.  What I want to exclude is specific endpoints like the Orchestration class and page-specific UI tests